components/automate-gateway/handler/compliance/reporting.go

```diff
@@ -48,6 +48,20 @@ func (a *Reporting) ListControlItems(ctx context.Context, in *reporting.ControlI
 	return out, nil
 }
 
+// should cover /nodecontrols/:reportid
```

Comment:
we're technically getting the controls from a report and not a node.  should this path be /reportcontrols instead of /nodecontrols?